### ISSUE TITLE
Push button to start playing a playlist/stream sample code

### ISSUE THREAD

**Issue by sassen** [MiczFlor/RPi-Jukebox-RFID#340]
Hi, is it possible to have a few dedicated push buttons for the most used playlists/streams on top of having the RFID card functionallity?

I only find instructions for GPIO buttons for play/stop, skip song, volume etc but nothing for playback of playlist/web radio stream.

**Comment by MiczFlor** [comment]
That could be hacked in. For one specific playlist. You would need to execute a bash script which imcludes the folder name. See top of this script for examples:
https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/master/scripts/rfid_trigger_play.sh
Use double quotes for a folder name with whitespaces
Looking forward to your working prototype. Please post it here. 
All the best, micz

**Comment by Bartman83** [comment]
Hallo Micz, i´m not good in coding, so i don`t know exactly which solution you prefer but i would try a workaround:
- I track a playlist with a card
- than i would add the following code in gpio-buttons.py  
`def def_play1():
    check_call("./rfid_trigger_play.sh --cardid=1234567890", shell=True)
play1 = Button(4,pull_up_True)
play1.when_pressed = def_play1`

I try it out when i bulid my next Box. Maybe someone think about my solution and give me a feedback.
Thank you so much for JukeBox4Kids!
Greets
Barti

**Comment by carver79** [comment]
Perfect - great someone did it. And a good solution for non-coders like me.
Going to try it the next days.
Thanks!

**Comment by MiczFlor** [comment]
Hi @Bartman83 
thanks for the code example. This is roughly what I had in mind with no time to write it :) If you could add this to the wiki, I would be very happy. Otherwise, I'll try to add it some time soon.
All the best, micz

**Comment by sassen** [comment]
Thank you so much!


Den fredag, november 23, 2018, 12:38 fm, skrev Micz Flor <notifications@github.com>:


Hi @Bartman83
thanks for the code example. This is roughly what I had in mind with no time to write it :) If you could add this to the wiki, I would be very happy. Otherwise, I'll try to add it some time soon.
All the best, micz

**Comment by MiczFlor** [comment]
Hi @Bartman83 
I added your example to the wiki:
https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/Hack-Playlist-triggered-by-GPIO-Button
thanks for the contribution!

**Comment by alanrodriguezdotme** [comment]
I've been able to get the jukebox controls working with the GPIO buttons, but I'm having trouble implementing the playlist controls. Here's what I have in my gpio-buttons.py:
![image](https://user-images.githubusercontent.com/8881195/49335494-24c27c00-f5a3-11e8-8005-a1e97b7d8b47.png)

I've tripled checked the GPIO connections are correct, and they are. I've even tried changing the buttons. No luck.
